src/dmd/globals.d

```diff
@@ -251,7 +250,6 @@ struct Global
     Param params;
     uint errors;            // number of errors reported so far
     uint warnings;          // number of warnings reported so far
-    FILE* stdmsg;           // where to send verbose messages
```

Comment:
I always hated that - :+1:

Reply:
GDC sends verbose messages to stderr because stdout is reserved for the use of `-pipe`, which was the rationale for having it in the first place.

If verbose messaging is encapsulated in `vmessage()` - and I implement my own versions of all these diagnostic functions anyway - then its use is no longer required.